kiwi/builder/disk.py

```diff
@@ -175,33 +173,15 @@ def create(self):
         Image types which triggers this builder are:
 
         * image="oem"
-        * image="vmx"
 
         :return: result
 
         :rtype: instance of :class:`Result`
         """
```

Comment:
This create method implementation is the root cause of all this refactor. I can't really understand why extra instances were required...